### DISCUSSION TITLE
How do I turn off the pprof feature in minio server

### DISCUSSION THREAD

**Discussion by heyuanchou** [minio/minio#18898] [Q&A] [answered]
How do I turn off the pprof feature in minio server

**Comment by heyuanchou** [comment]
pprof is a performance analysis tool for the Go language
I wonder if minio server is useful for this
and
How do I turn off the pprof feature in minio server

**Reply by klauspost** [reply, marked as answer]
pprof is used for `mc support profile`. It is only running when enabled (except memory profiling, which tracks lifetime allocs). There is no way to turn it off since there is no reason to turn it off.

**Reply by heyuanchou** [reply]
ok，thank you very much
